dev/ledger.md

```diff
@@ -348,12 +354,16 @@ A payment transaction additionally has the following fields:
  - The _closing address_ $I_0$, which is an optional address that collects all
    remaining funds in the account after the transfer to the receiver.
 
-A key registration transaction additionally has the following field:
+A key registration transaction additionally has the following fields:
 
  - The new participation keys $\pk$, which is an optional set of account
    participation keys to be registered to the account.  If unset, the
    transaction deregisters the account's keys.
 
+ - An optional (boolean) flag $\nonpart$ which, when deregistering keys,
+   specifies whether to mark the account offline (if $\nonpart$ is false)
+   or nonparticipatory (if $\nonpart$ is true). If absent, $\nonpart$ is considered false.
```

Comment:
> If absent, $\nonpart$ is considered false.

I think by virtue of our canonical encoding, the field will be absent if and only if nonpart is false. (Thus I'd remove this line.)